src/writer/proposal.ts

```diff
@@ -187,6 +187,7 @@ export async function action(body, ipfs, receipt, id): Promise<void> {
   /* Store the proposal in dedicated table 'proposals' */
   const spaceSettings = await getSpace(space);
 
+  if (!spaceSettings) return Promise.reject('unknown space');
```

Comment:
This one is already check in `verify`, and redundant

Reply:
```suggestion
```